src/pages/Dashboard/index.tsx

```diff
@@ -38,28 +39,43 @@ const DashboardPage: React.FC = () => {
     setValue(value);
   };
 
+  const { data: clientsData } = useGetClients();
+
   return (
     <div className="flex flex-col items-center">
-      <div className="mb-4 mt-[63px] flex w-[1048px] justify-between">
-        <SearchBar
-          value={value}
-          onChange={handleChange}
-          placeholder="고객사명을 입력해주세요."
-          bgColor="white"
-        />
-        <button
-          type="button"
-          className="flex h-10 gap-1 rounded border-[0.5px] border-primary-200 bg-surface-secondary p-2 pl-3 text-md font-semibold text-primary-700"
-          onClick={handleAddModalOpen}
-        >
-          고객사 추가하기
-          <AddCircleFillIcon className="fill-primary-500" />
-        </button>
-      </div>
-      <div className="grid w-[1048px] grid-cols-4 gap-3">
-        <MonitoringCard name="한솥" manager="이현수" />
-      </div>
-      {/* <ClientNotFound handleAddModalOpen={handleAddModalOpen} /> */}
+      {!clientsData || clientsData?.length === 0 ? (
+        <ClientNotFound handleAddModalOpen={handleAddModalOpen} />
+      ) : (
+        <>
+          <div className="mb-4 mt-[63px] flex w-[1048px] justify-between">
+            <SearchBar
+              value={value}
+              onChange={handleChange}
+              placeholder="고객사명을 입력해주세요."
+              bgColor="white"
+            />
+            <button
+              type="button"
+              className="flex h-10 gap-1 rounded border-[0.5px] border-primary-200 bg-surface-secondary p-2 pl-3 text-md font-semibold text-primary-700"
+              onClick={handleAddModalOpen}
+            >
+              고객사 추가하기
+              <AddCircleFillIcon className="fill-primary-500" />
+            </button>
```

Comment:
[변경 요청]
```suggestion
            <Button
              style="tonal"
              className="flex items-center h-10 gap-1 p-2 pl-3"
              onClick={handleAddModalOpen}
            >
              고객사 추가하기
              <AddCircleFillIcon className="fill-primary-500" />
            </Button>
```
이렇게 바꿔주세요